pyOCD/tools/pyocd.py

```diff
@@ -22,8 +22,9 @@
 
 import os
 import pyOCD
-from pyOCD.board import MbedBoard
-from pyOCD.target import target_kinetis
+from ..board import MbedBoard
+from ..target import target_kinetis
+from .. import __version__
```

Comment:
I've reverted the change and tried my best to fix import according to pep8 with absolute imports.

I failed to find the relevant entry in the pep which support the use of absolute imports in the case above.